### DISCUSSION TITLE
Run in webgpu

### DISCUSSION THREAD

**Discussion by pdufour** [NVIDIA-NeMo/NeMo#10889] [Q&A]
Does anyone know if it's possible to run https://huggingface.co/nvidia/canary-1b via WebGPU? Possibly via https://github.com/xenova/transformers.js/ or https://github.com/mlc-ai/web-llm. I understand people have done it with whisper (https://huggingface.co/spaces/Xenova/whisper-webgpu) but wanted to try out canary.

I think maybe if I export it a ONNX version I can use it with transformers.js? https://docs.nvidia.com/nemo-framework/user-guide/latest/nemotoolkit/core/export.html

**Comment by Tejaswgupta** [comment]
@pdufour did you get an answer for this?
